pages/boards/components/BestBoardCard.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+import styles from './BestBoardCard.module.css';
+import medalSvg from '@/src/assets/ic_medal.svg';
+import heardSvg from '@/src/assets/ic_heart.svg';
+import { Board } from '@/src/apis/boardTypes';
+
+interface BestBoardCardProps
+  extends Pick<
+    Board,
+    'title' | 'image' | 'writer' | 'likeCount' | 'createdAt'
+  > {}
```

Comment:
P2:
추가로 정의할 타입이 있지 않다면 아래처럼 하시는 것을 추천드립니다.
```suggestion
type BestBoardCardProps = Pick<
    Board,
    'title' | 'image' | 'writer' | 'likeCount' | 'createdAt'
  >;
```